object.rb

```diff
@@ -172,3 +172,65 @@ def Float(arg, exception: true)
     Primitive.rb_f_float(arg, exception)
   end
 end
+
+class TrueClass
+  #
+  # call-seq:
+  #   true.to_s   ->  "true"
+  #
+  # The string representation of <code>true</code> is "true".
+  #
+  def to_s
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'rb_cTrueClass_to_s'
```

Comment:
I was wondering if we can simply write `-"true"` here, but then I guess source encoding needs to be US-ASCII to keep the encoding. (just leaving my thought, not suggesting any change)

Reply:
-"true" was slow than use `rb_cTrueClass_to_s`, so use `rb_cTrueClass_to_s`.